### PR TITLE
More leftovers

### PR DESCRIPTION
**Pull request by unho**
These are more leftovers from the move to pootle_fs and the drop of old stats infrastructure.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,9 @@
 !/pootle/translations/tutorial/
 !/pootle/translations/templates/
 
+# /pootle/pootle_fs/
+/pootle/pootle_fs/*/
```
Comment (phlax) [inline .gitignore]: i think this needs to be called pootle_fs_working or similar and be more hidden - fail here is users a) touching the files b) adding stuff here

File: docs/server/commands.rst [hunk 2]
```diff
@@ -1082,6 +1039,26 @@ Deprecated commands
 The following are commands that have been removed or deprecated:
 
 
+.. django-admin:: refresh_stats
+
+refresh_stats
+^^^^^^^^^^^^^
+
+.. deprecated:: 2.8
```
Comment (phlax) [inline docs/server/commands.rst]: they are not deprecated - they are removed

File: docs/server/settings.rst [hunk 3]
```diff
@@ -474,9 +474,8 @@ Translation environment configuration settings.
 
   Adding a custom function allows you to alter how words are counted.
 
-  .. warning:: Changing this function requires that you run
-     :djadmin:`refresh_stats --calculate-wordcount <refresh_stats>` to
-     recalculate the associated wordcounts.
+  .. warning:: Changing this function requires to recalculate the associated
```
Comment (phlax) [inline docs/server/settings.rst]: s/to/that you/

File: docs/server/upgrading.rst [hunk 4]
```diff
@@ -193,23 +193,21 @@ data upgrade by running. This needs to be done in a few steps:
...
-Refreshing checks and stats
----------------------------
+Refreshing checks
+-----------------
 
-You must now update the translation checks and populate the Redis cache with
-statistical data. You will need to have an :ref:`RQ worker running 
-<installation#running-rqworker>` to complete this.
+You must now update the translation checks. You will need to have an
+:ref:`RQ worker running <installation#running-rqworker>` to complete this.
```
Comment (phlax) [inline docs/server/upgrading.rst]: i dont think you do need an rq worker running any longer

File: docs/server/upgrading.rst [hunk 5]
```diff
@@ -193,23 +193,21 @@ data upgrade by running. This needs to be done in a few steps:
...
 
    (env) $ pootle calculate_checks
-   (env) $ pootle refresh_stats
 
 This command will dispatch jobs to the RQ worker and may take some time.
 
-If you wish to run :djadmin:`calculate_checks` and :djadmin:`refresh_stats` in
+If you wish to run :djadmin:`calculate_checks` ain
 the foreground without using the RQ worker you can use the :option:`--no-rq`
 option.
```
Comment (phlax) [inline docs/server/upgrading.rst]: not sure if this matters

File: docs/server/upgrading.rst (hunk 4 above)
Comment (unho) [inline docs/server/upgrading.rst]: Lets revisit this later.

File: docs/server/commands.rst (hunk 2 above)
Comment (unho) [inline docs/server/commands.rst]: Just noticed there is indeed one command using `removed`. Will adjust.

File: docs/server/settings.rst (hunk 3 above)
Comment (unho) [inline docs/server/settings.rst]: Changed.

File: .gitignore (hunk 1 above)
Comment (unho) [inline .gitignore]: I don't this is a problem. Some similar would happen with `/pootle/translations/` IMHO.

File: docs/server/upgrading.rst (hunk 4 above)
Comment (phlax) [inline docs/server/upgrading.rst]: k

File: .gitignore (hunk 1 above)
Comment (phlax) [inline .gitignore]: that **is** the problem exactly

Comment (unho) [inline .gitignore]: Anyway changed.

**Comment by phlax** [comment]
lgtm

**Pull request merged**